src/views/FacilityManagement.vue

```diff
@@ -69,10 +106,92 @@ export default defineComponent({
     IonHeader,
     IonIcon,
     IonItem,
+    IonLabel,
+    IonNote,
     IonPage,
     IonTitle,
     IonToolbar
   },
+  computed: {
+    ...mapGetters({
+      userProfile: 'user/getUserProfile',
+      instanceUrl: 'user/getInstanceUrl',
+    })
+  },
+  data() {
+    return {
+      warehouseFacilitiesCount: 0,
+      physicalStoreFacilitiesCount: 0,
+      brokeringQueueOrderCount: 0,
+      backorderParkingOrderCount: 0,
+      pickupFacilitiesCount: 0,
+      onlineFacilitiesCount: 0
+    }
+  },
+  async ionViewWillEnter() {
+    await this.fetchFacilityTypeCount()
+    await this.fetchParkingsOrderCount()
+    await this.fetchGroupsFacilitiesCount()
+  },
+  methods: {
+    async fetchFacilityTypeCount() {
+      const warehouseTypePayload = {
+        "inputFields": {
+          'parentFacilityTypeId': 'DISTRIBUTION_CENTER',
+        },
+        "entityName": "FacilityAndProductStore",
+        "distinct": "Y",
+        "fieldList": ['facilityId', 'facilityName', 'facilityTypeId'],
+        "viewSize": 1
+      }
+  
+      const physicalStoreTypePayload = {
+        "inputFields": {
+          'parentFacilityTypeId': 'PHYSICAL_STORE',
+        },
+        "entityName": "FacilityAndProductStore",
+        "distinct": "Y",
+        "fieldList": ['facilityId', 'facilityName', 'facilityTypeId'],
+        "viewSize": 1
+      }
+
+      try {
+        const responses = await Promise.allSettled([
```

Comment:
I think instead of making two api calls, we can simply make a single api call and then filter the results here on the basis of type.